wlauto/workloads/applaunch/__init__.py

```diff
@@ -117,11 +123,20 @@ def setup(self, context):
         self.device.clear_logcat()
 
     def run(self, context):
-        self.device.execute('sh {}'.format(self.device_script_file), timeout=300)
+        self.device.execute('sh {}'.format(self.device_script_file), timeout=300, as_root=True)
```

Comment:
This change limits this test to only rooted devices. Since root is only required for io stress, please change to

     self.device.execute('sh {}'.format(self.device_script_file), timeout=300, as_root=self.io_stress)